[2023-12-10T17:53:58.262+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test.merge_df manual__2023-12-10T12:53:40.707062+00:00 [queued]>
[2023-12-10T17:53:58.274+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test.merge_df manual__2023-12-10T12:53:40.707062+00:00 [queued]>
[2023-12-10T17:53:58.275+0500] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-10T17:53:58.299+0500] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): merge_df> on 2023-12-10 12:53:40.707062+00:00
[2023-12-10T17:53:58.303+0500] {standard_task_runner.py:57} INFO - Started process 7425 to run task
[2023-12-10T17:53:58.315+0500] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'test', 'merge_df', 'manual__2023-12-10T12:53:40.707062+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp63nrn7ye']
[2023-12-10T17:53:58.317+0500] {standard_task_runner.py:85} INFO - Job 34: Subtask merge_df
[2023-12-10T17:53:58.384+0500] {task_command.py:416} INFO - Running <TaskInstance: test.merge_df manual__2023-12-10T12:53:40.707062+00:00 [running]> on host xypma-VirtualBox
[2023-12-10T17:53:58.476+0500] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='test' AIRFLOW_CTX_TASK_ID='merge_df' AIRFLOW_CTX_EXECUTION_DATE='2023-12-10T12:53:40.707062+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-10T12:53:40.707062+00:00'
[2023-12-10T17:53:58.477+0500] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-10T17:53:58.478+0500] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/xypma/lab3/MLOps_lab3/ && python /home/xypma/lab3/MLOps_lab3/scripts/merge_df.py']
[2023-12-10T17:53:58.490+0500] {subprocess.py:86} INFO - Output:
[2023-12-10T17:53:59.961+0500] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-10T17:54:00.016+0500] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test, task_id=merge_df, execution_date=20231210T125340, start_date=20231210T125358, end_date=20231210T125400
[2023-12-10T17:54:00.071+0500] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-10T17:54:00.119+0500] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
